src/util.cpp

```diff
@@ -889,6 +889,97 @@ QSize CMinimumStackedLayout::sizeHint() const
 * Other Classes                                                                *
 \******************************************************************************/
 // Network utility functions ---------------------------------------------------
+bool NetworkUtil::ParseNetworkAddressName ( QString strAddress, QHostAddress& InetAddr, bool bEnableIPv6 )
```

Comment:
I'm not keen on this method name.  It doesn't explain that it's the one place we turn any string into an IP address.  I think I'd prefer `ParseNetworkAddressString`.

Reply:
Ok, changed.